Dockerfile

```diff
@@ -24,6 +24,19 @@ RUN git config --global user.email "meshery@layer5.io"
 RUN git config --global user.name "meshery"
 RUN git clone --depth=1 https://github.com/layer5io/wrk2 && cd wrk2 && make
 
+FROM alpine:3.14 as seed_content
+RUN apk add --no-cache curl
+WORKDIR /
+RUN curl -L -s `curl -s https://api.github.com/repos/layer5io/wasm-filters/releases/latest | grep "browser_download_url" | cut -d : -f 2,3 | tr -d '"'` -o wasm-filters.tar.gz \
+    && mkdir -p /seed_content/filters/binaries \
+    && tar xzf wasm-filters.tar.gz --directory=/seed_content/filters/binaries
+
+RUN curl -L -s https://github.com/service-mesh-patterns/service-mesh-patterns/tarball/master -o service-mesh-patterns.tgz \
+    && mkdir service-mesh-patterns \
+    && mkdir -p /seed_content/patterns \
+    && tar xzf service-mesh-patterns.tgz --directory=service-mesh-patterns \
+    && mv service-mesh-patterns/*/samples/* /seed_content/patterns/

```

Comment:
@DelusionalOptimist @Revolyssup where is the seed content being stored long-term on the filesystem? What about content that has been uploaded to the Local Provider after server initialization?